ansible/roles/formplayer/templates/application.properties.j2

```diff
@@ -49,8 +49,12 @@ spring.jpa.hibernate.ddl-auto
 
 smtp.host={{ localsettings.EMAIL_SMTP_HOST }}
 smtp.port={{ localsettings.EMAIL_SMTP_PORT }}
-smtp.username={{ localsettings.EMAIL_LOGIN }}
-smtp.password={{ localsettings.EMAIL_PASSWORD|default('dummy') }}
+smtp.username={{ EMAIL_LOGIN }}
+{% if EMAIL_PASSWORD or EMAIL_PASSWORD == 'None' %}
```

Comment:
`EMAIL_PASSWORD == 'None'` seems redundant here. Is there some strange ansible evaluation going on that would make the string `'None'` evaluate to false?

Reply:
Oops, I meant to remove this, sorry.

This and a couple other variables were set in various vault files to `None`, which yaml interprets not as python `None`, but as the string `'None'`. I ended up changing these to use `null` as clearly intended, but apparently forgot to remove this workaround I played with when I was trying to figure out what was going on. I'll remove.